src/richie/apps/courses/templates/courses/cms/fragment_course_content.html

```diff
@@ -85,15 +83,11 @@ <h2 class="section__title course-detail__content__row__title course-detail__cont
   <h2 class="section.title course-detail__content__row__title course-detail__content__team__title">
     {% trans 'Course team' %}
   </h2>
-  {% with header_level=3 %}
-    {% if page|is_empty_placeholder:"course_team" %}
-      <p>{% trans 'Who are the teachers in the course team?' %}</p>
-    {% else %}
-      <div class="section__items course-detail__content__team__items">
-        {% page_placeholder "course_team" page %}
-      </div>
-    {% endif %}
-  {% endwith %}
+    <div class="section__items course-detail__content__team__items">
```

Comment:
ditto with:
```django
{% if current_page.publisher_is_draft or not current_page|is_empty_placeholder:"course_team" %}
```

Reply:
Same with this one : https://github.com/openfun/richie/blob/5bd8614323e5ea791f42037f800e1380baa5e9e5/src/richie/apps/courses/templates/courses/cms/fragment_course_content.html#L81